fluentd/generate_throttle_configs.rb

```diff
@@ -0,0 +1,108 @@
+require 'json'
+require 'date'
+require 'logger'
+
+# or should this be stderr?  -- we only want to print out warns?
```

Comment:
Does anything in the fluentd container *have* any output that isn't a log message? I'm inclined to leave it stdout unless there's output it would be useful to distinguish from log messages.